tests/test_feature_detection.py

```diff
@@ -0,0 +1,41 @@
+"""Unit test related to feature detection model instanciation
```

Comment:
I would have written "Unit test related to the feature detection model"

I don't think that the word `instanciation` is correct.

Reply:
`instanciation` word has been removed.